database_mariadb/mariaDB-scripts/InsertData.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO automates (unit_id, sensor_id, check_date, sensor_type, cistern_temperature, ambient_temperature, cistern_milk_weight, finish_product_weight, ph, kplus, nacl, salmonella_lvl, ecoli_lvl, listeria_lvl) 
```

Comment:
This won't be need when the collecteur is working, shoudl we remove it before the merge ?